src/components/comment/comment.scss

```diff
@@ -0,0 +1,131 @@
+@use 'sass:math';
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/font-weight';
+@use '../../compiled/tokens/scss/size';
+@use '../../mixins/headings';
+@use '../../mixins/theme';
+
+.c-comment {
+  display: grid;
+  grid-column-gap: size.$spacing-gap-inline-medium;
+  grid-row-gap: size.$rhythm-condensed;
+  grid-template-areas:
+    'object header'
+    'thread-line content'
+    'thread-line footer';
+  // We set the object column size directly (instead of relying on `auto`)
+  // because the container for child comments will be setting a negative margin
+  // based on this value, and we don't want the layout to break if the wrong
+  // avatar size is set in the HTML.
+  grid-template-columns: #{size.$square-avatar-small} 1fr;
+  grid-template-rows: minmax(0, auto) 1fr minmax(0, auto);
+}
+
+/// Display a vertical line if the comment contains a reply thread to connect
+/// those comments visually with their parent.
+.c-comment--thread::after {
+  border-radius: size.$border-radius-full;
+  content: '';
+  display: block;
+  grid-area: thread-line;
+  height: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  width: size.$edge-medium;
+
+  @include theme.styles() {
+    background-color: color.$base-gray-light;
+  }
+
+  @include theme.styles(dark) {
+    background-color: color.$base-blue-darker;
+  }
+}
+
+/// Named object for consistency with the Media component.
+.c-comment__object {
+  grid-area: object;
+}
+
+.c-comment__header {
+  // We generally want comment contents to align to the start/top, but the
+  // the heading looks better center-aligned with the avatar.
```

Comment:
@Paul-Hebert I've addressed this feedback, but I removed a different "the" because there was plenty of room below the 80 character threshold for that line.